rosie/README.md

```diff
@@ -45,7 +45,7 @@ For example, if you want to run `chamber_of_deputies` module, you should run thi
 $ python rosie.py run chamber_of_deputies
 ```
 
-The toolbox expects to find Amazon credentials in a `config.ini` file. If You get an error of missing `config.ini` you can just copy the `config.ini.example` to a `config.ini` and everything will run smoothly (don't bother about `AccessKey` and `SecretKey` unless you're planning to upload files to S3).
+The toolbox expects to find Amazon credentials in a `.env` file. If You get an error of missing `.env` you can just copy the `.env.sample` to a `.env` and everything will run smoothly (don't bother about `AccessKey` and `SecretKey` unless you're planning to upload files to S3).
```

Comment:
In the `.env` file there's no `AccessKey` and `SecretKey` anymore. If I'm not wrong they're called `AMAZON_ACCESS_KEY` and `AMAZON_SECRET_KEY`.